tests/editor_utils_test.py

```diff
@@ -1923,3 +1923,152 @@ def test_client_compatibility(self):
+                        },
+                        "2": {
+                            "blocks": [
+                                {
+                                    "key": "0b83550e-1fe6-4e67-9b03-2dc3245aae18",
+                                    "text": "(10,2704)",
+                                    "type": "unstyled",
+                                    "depth": 0,
+                                    "inlineStyleRanges": [],
+                                    "entityRanges": [],
+                                    "data": {},
+                                }
+                            ],
+                            "entityMap": {},
+                        },
+                    },
+                    "2": {
+                        "0": {
+                            "blocks": [
+                                {
+                                    "key": "1216aeed-af9e-4811-9467-6bbb24bbbda5",
+                                    "text": "Sveitsiske franc\xa0(CHF)",
+                                    "type": "unstyled",
+                                    "depth": 0,
+                                    "inlineStyleRanges": [],
+                                    "entityRanges": [],
+                                    "data": {},
+                                }
+                            ],
+                            "entityMap": {},
+                        },
+                        "1": {
+                            "blocks": [
+                                {
+                                    "key": "3d316782-b1bf-40d8-8fd2-bc1492f187c4",
+                                    "text": "",
+                                    "type": "unstyled",
+                                    "depth": 0,
+                                    "inlineStyleRanges": [],
+                                    "entityRanges": [],
+                                    "data": {},
+                                }
+                            ],
+                            "entityMap": {},
+                        },
+                        "2": {
+                            "blocks": [
+                                {
+                                    "key": "b33559c4-534d-4df1-b8da-5e3c26727e47",
+                                    "text": "(1000,1363)",
+                                    "type": "unstyled",
+                                    "depth": 0,
+                                    "inlineStyleRanges": [],
+                                    "entityRanges": [],
+                                    "data": {},
+                                }
+                            ],
+                            "entityMap": {},
+                        },
+                    },
+                },
+            },
+            "entity": {"type": "TABLE"},
+            "children": " ",
+        }
+
+        item = DraftJSHTMLExporter.render_table(self, data)
+        self.assertFalse(item.items(), [])
```

Comment:
why would it be false? I still don't get why this won't render actual table.

Reply:
We can make it `self.assertEqual(item.items(), [])`.

Reply:
>  render actual table.

By actual table, you mean it should return `<table></table>`